Cargo.toml

```diff
@@ -16,15 +16,15 @@ include = [
 ]
 
 [dependencies]
-bytes = "1.4.0"
+bytes = "1.6.0"
 digest = "0.10.7"
-fastrand = "2.0.0"
+fastrand = "2.0.2"
 hex = "0.4.3"
-reqwest = { version = "0.11.18", features = [ "blocking" ] }
+reqwest = { version = "0.11.27", features = [ "blocking", "native-tls-vendored" ] }
```

Comment:
Why `native-tls-vendored` ?

If you need this, I think better solution would be to add optional feature `native-tls-vendored` which enables this, instead of making it default.

Reply:
Ah, I'll remove it. Forgot to remove it after testing with workspace flags.